includes/admin/donors/donor-actions.php

```diff
@@ -240,104 +240,6 @@ function give_donor_save_note( $args ) {
 
 add_action( 'give_add-donor-note', 'give_donor_save_note', 10, 1 );
 
-/**
- * Delete a donor.
- *
- * @param array $args The $_POST array being passed.
- *
- * @since 1.0
- *
- * @return int Whether it was a successful deletion.
- */
-function give_donor_delete( $args ) {
```

Comment:
@mehul0810 can you deprecate this function so that it doesn't result it fatal errors.

Reply:
@DevinWalker Added deprecate functions.

Reply:
@mehul0810 I don't see a deprecated `give_delete_donor` function, I see deprecated _actions_. I'm asking that the function be deprecated as well in case anyone is using it directly.

Reply:
@DevinWalker I've added deprecated `give_delete_donor` as well as `give_donor_delete` fn and if the user calls that function then deprecated notice will be displayed as well as the function is backward compatible to ensure that the functionality is not broken.